src/FE/src/app/pages/profile-page/profile-page.component.ts

```diff
@@ -222,4 +222,14 @@ export class ProfilePageComponent implements OnInit {
     let hourlyRate = teamProfile.allocatedCost! / teamProfile.allocatedHours!
     return hourlyRate * profile.hoursPerDay!;
   }
+
+  async remove(row: any) {
+    let result = await this.teamsService.deleteTeamProfile(row.id);
+    this.datasource.data = this.datasource.data.filter(item => item !== row);
```

Comment:
Kunne vi ændre navngivning, så det ikke er item, men hvad det ér, tror teamProfile i dette tilfælde?

eventuelt giv row in type som er hvad det faktisk er også?